js/src/setup-mc/setup-stepper/setup-paid-ads/paid-ads-setup-sections.js

```diff
@@ -108,14 +124,18 @@ export default function PaidAdsSetupSections( {
 	}
 
 	const initialValues = {
-		amount: paidAds.amount,
+		amount: clientSession.getCampaign()?.amount || dailyBudget,
 	};
 
 	return (
 		<Form
 			initialValues={ initialValues }
 			onChange={ ( _, values, isValid ) => {
 				setPaidAds( { ...paidAds, ...values, isValid } );
+
+				if ( isValid ) {
+					clientSession.setCampaign( values );
```

Comment:
@ankitrox The campaign is already being updated in local storage here: 
https://github.com/woocommerce/google-listings-and-ads/pull/2552/files#diff-6b9f6824ae1208e70b059bf81217131f2d1c42fad1664514c2280500e8e1a7b6R114

Reply:
Outdated with latest changes.